web/src/shared/components/Graphs/DonutChart.tsx

```diff
@@ -9,10 +9,14 @@ export interface IDonutChartData {
 }
 
 export interface IDonutChartProps {
+    id?: string | undefined
```

Comment:
The question mark implies undefined here (and below)
```suggestion
    id?: string
```

Reply:
Excellent point :-)